src/main/webapp/isolate/request/scripts/tickets.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {TicketFactory} from './TicketFactory.js';
+import {removeFromDOM} from './DOMUtils.js';
+
+/* Create a TicketFactory to create tickets during the session */
+const ticketFactory = new TicketFactory();
+
+/**
+ * Delete a ticket from the request submission form.
+ * @param {!HTMLElement} ticket The ticket to be deleted from the form.
+ * @throws {TypeError} Will throw a TypeError if ticket is not viable for
+ *     deletion.
+ */
+export const deleteTicket =
+    (ticket) => {
+      const TYPE_MISMATCH_PREFIX = `Ticket to be removed from form must not be`;
+
+      if (ticket === null) {
+        throw (TypeError(`${TYPE_MISMATCH_PREFIX} null.`));
+      }
+
+      else if (typeof ticket === 'undefined') {
+        throw (TypeError(`${TYPE_MISMATCH_PREFIX} undefined.`));
+      }
```

Comment:
why there is this space between if and else if?

Reply:
I've fixed this now https://github.com/googleinterns/step226-2020/pull/5/commits/4f0530b45e59094d94ed6c539fa231e58cd2ab97